src/main/java/com/jcabi/aether/Aether.java

```diff
@@ -89,6 +91,10 @@
  * @checkstyle ClassFanOutComplexity (500 lines)
  * @see <a href="http://sonatype.github.com/sonatype-aether/apidocs/overview-tree.html">Aether 1.13.1 JavaDoc</a>
  * @see Classpath
+ * @todo #11:30min Create an integration test for reading proxy information
+ * from maven settings.xml file that will mock an HTTP server
```

Comment:
@adko-pl  we need a space here, before the 'f' and also on the next 2 lines (before 'u' and 'a')